spec/features/employer/employer_can_login_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
@jmejia: I think we should add a test for multitenancy - i.e. a user can't see/access the employer dashboard. Does that make sense?